lua/tardis/parts/legacy/legacy_blacksticks.lua

```diff
@@ -7,5 +7,11 @@ PART.Model = "models/drmatt/tardis/blacksticks.mdl"
 PART.AutoSetup = true
 PART.Collision = true
 PART.Animate = true
+PART.BypassIsomorphic = false
+
+function PART:Use(ply)
+    local result = self.exterior:ToggleCloak() or false
```

Comment:
This function does not exist, and will just throw an error message, why is it here?